src/app/resperson/index.njk

```diff
@@ -56,10 +56,10 @@
                       <button id="edit-edit_reponsiblePerson-{{ loop.index }}" class="govuk-link btn-link" 
                         title="Edit details for {{ resPerson.responsibleGivenName }} {{ resPerson.responsibleSurname }}" 
                         formaction="resperson/edit" formmethod="get"
-                        name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleGivenName}}
+                        name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleSurname}}
                       </button>
                     </td>
-                    <td class="govuk-table__cell" scope="row">{{resPerson.responsibleSurname}}</td>
+                    <td class="govuk-table__cell" scope="row">{{resPerson.responsibleGivenName}}</td>
```

Comment:
Nit: `scope="row"` should be on the surname element than the resperson given name if it is to be used.